### PR TITLE
Improve error handling when config not found

### LINKED ISSUE #18: Improve error handling when configuration file is not found
The error message should attempt to explain to the user how to [get started](https://github.com/skanehira/github-tui?tab=readme-ov-file#settings) with the configuration file(required), as it does not provide a command to generate one automatically.

### Screenshoot

image

### PR DESCRIPTION
**Pull request by luisnquin**
Closes #18.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -30,32 +32,37 @@ func Init() {
...
+		log.Fatalf("Could not find configuration file, %s", readThisMessage)
+	}
 
 	var conf struct {
 		GitHub github `yaml:"github"`
 	}
 
 	if err := yaml.Unmarshal(b, &conf); err != nil {
-		log.Fatal(err)
+		log.Fatalf("cannot deserialize config file: %s", err.Error())
```
Comment (skanehira) [inline config/config.go:59]: I think we do not need call `Error()` method
```suggestion
		log.Fatalf("cannot deserialize config file: %s", err)
```

**Review by skanehira — CHANGES_REQUESTED** [review]
Thanks :)
I left a comment

**Comment by luisnquin** [comment]
Done @skanehira.

**Comment by skanehira** [comment]
LGTM!

**Pull request merged**